server/index.js

```diff
@@ -1,12 +1,68 @@
+
+/* eslint-disable no-console */
```

Comment:
Please do not turn this rule off. Use `console.error()` if you need to log errors.